core/sys/windows/gdi32.odin

```diff
@@ -62,5 +62,11 @@ foreign gdi32 {
 	ChoosePixelFormat :: proc(hdc: HDC, ppfd: ^PIXELFORMATDESCRIPTOR) -> c_int ---
 	SwapBuffers :: proc(HDC) -> BOOL ---
 
+	SetDCBrushColor :: proc(hdc: HDC, color: COLORREF) -> COLORREF ---
+	GetDCBrushColor :: proc(hdc: HDC) -> COLORREF ---
 	PatBlt :: proc(hdc: HDC, x, y, w, h: c_int, rop: DWORD) -> BOOL ---
 }
+
+RGB :: proc(r, g, b: u8) -> COLORREF {
```

Comment:
I'd perhaps gone with this:
```cpp
RGB :: proc(rgb: [3]u8) -> COLORREF {
    res: [4]u8 = {0, rgb.b, rgb.g, rgb.r} // ABGR
    return transmute(COLORREF)res
}
```
And also put a comment on COLORREF that it's packed as ABGR.